data-model/src/main/scala/za/co/absa/enceladus/model/properties/propertyType/package.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.model.properties
+
+import com.fasterxml.jackson.annotation.JsonSubTypes.Type
+import com.fasterxml.jackson.annotation.{JsonSubTypes, JsonTypeInfo}
+
+import scala.util.{Failure, Success, Try}
+
+package object propertyType {
+
+  @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "_t")
+  @JsonSubTypes(Array(
+    new Type(value = classOf[StringPropertyType], name = "StringPropertyType"),
+    new Type(value = classOf[EnumPropertyType ], name = "EnumPropertyType ")
```

Comment:
```suggestion
    new Type(value = classOf[EnumPropertyType ], name = "EnumPropertyType")
```

Reply:
Yup, fixed that with the suggestedValue-fix, too.